lib/fortnox/api/models/row/entity.rb

```diff
@@ -0,0 +1,66 @@
+require "fortnox/api/models/entity/base"
+
+module Fortnox
+  module API
+    module Entities
+      class Row < Fortnox::API::Entities::Base
```

Comment:
Yeah, I agree. Keep it in models, but use the same basic principle as with the attribute classes.